aries-site/src/examples/components/avatar/AvatarDataTableIconExample.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {
+  Avatar,
+  Box,
+  DataTable,
+  Image,
+  ResponsiveContext,
+  Text,
+} from 'grommet';
+import company from '../../../data/mockData/company.json';
+
+export const AvatarDataTableIconExample = ({ bestPractice = true }) => {
+  const size = React.useContext(ResponsiveContext);
+  const [selected, setSelected] = React.useState([]);
+
+  return (
+    <>
+      <Box width="medium" height={{ max: 'large' }} overflow="auto">
+        <DataTable
+          data={company}
+          primaryKey="name"
+          columns={[
+            {
+              property: 'name',
+              header: 'Name',
+              render: datum => (
+                <>
+                  {bestPractice ? (
+                    <Box alignContent="center" gap="small" direction="row">
+                      <Avatar
+                        src={datum.src}
+                        align="undefined"
+                        round="medium"
+                        size="small"
+                      />
+                      <Text>{datum.name}</Text>
+                    </Box>
+                  ) : (
+                    <Box
+                      alignSelf="start"
+                      align="center"
+                      gap="small"
+                      alignContent="center"
+                      direction="row"
+                    >
+                      <Image src={datum.src} />
```

Comment:
I was expecting only the ctera/oracle logos which are not naturally equal dimensions to be wider here as opposed to some ones that are taller too.
```suggestion
                      <Box height="small">
                         <Image src={datum.src} />
                      </Box>
```
<img width="572" alt="Screen Shot 2022-10-26 at 9 22 37 AM" src="https://user-images.githubusercontent.com/12522275/198081263-c8e3ae00-c650-4e35-a5d0-aabe47f2c991.png">

Reply:
I think this is still okay since it still gets the point across, but if we do have other logos such as Ctera that are naturally wider then I agree we should use those instead. 

Also, I don't think thats the correct logo for Suse

Reply:
looking into this